src/nat/middleware/cache_middleware.py

```
@@ -0,0 +1,254 @@
+
+        Args:
+            value: The input value to serialize.
+
+        Returns:
+            String representation of the input, or None if serialization
+            fails.
+        """
+        try:
+            # Try JSON serialization first for best results
+            return json.dumps(value, sort_keys=True, default=str)
+        except Exception:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)